msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -122,37 +152,164 @@ The query parameters are:
+
+#### Generate a deep link to the scheduling dialog
+
+Use the following format for a deep link that you can use in a bot, Connector, or message extension card:
+`https://teams.microsoft.com/l/meeting/new?subject=<meeting subject>&startTime=<date>&endTime=<date>&content=<content>&attendees=<user1>,<user2>,<user3>,...`
+
+Example: `https://teams.microsoft.com/l/meeting/new?subject=test%20subject&attendees=joe@contoso.com,bob@contoso.com&startTime=10%2F24%2F2018%2010%3A30%3A00&endTime=10%2F24%2F2018%2010%3A30%3A00&content=​​​​​​​test%3Acontent​​​​​​​​​​​​​​`
+
+> [!NOTE]
+> The search parameters don't support `+` signal in place of whitespace (` `). Ensure your uri encoding code returns `%20` for spaces for example, `?subject=test%20subject` is good, but `?subject=test+subject` is bad.
+
+The query parameters are:
+
+* `attendees`: The optional comma-separated list of user IDs representing the attendees of the meeting. The user performing the action is the meeting organizer. The User ID field currently only supports the Azure AD UserPrincipalName, typically an email address.
+* `startTime`: The optional start time of the event. This should be in [long ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601), for example *2018-03-12T23:55:25+02:00*.
+* `endTime`: The optional end time of the event, also in ISO 8601 format.
+* `subject`: An optional field for the meeting subject.
+* `content`: An optional field for the meeting details field.
+
+> [!NOTE]
+> Currently, specifying the location isn't supported. You must specify the UTC offset, it means time zones when generating your start and end times.
+
+To use this deep link with your bot, you can specify this as the URL target in your card's button or tap action through the `openUrl` action type.
+
+### Open an app install dialog
+
+You can use a deep link to open an app install dialog from your Teams app, as shown in the following code.
 
+# [TeamsJS v2](#tab/teamsjs-v2)
+
+```javascript
+// Open an app install dialog from your tab
+if(appInstallDialog.isSupported()) {
+    const dialogPromise = appInstallDialog.openAppInstallDialog({ appID: <appId>});
+    dialogPromise.
+      then((result) => {/*Successful operation*/}).
+      catch((error) => {/*Unsuccessful operation*/});
+}
+else { /* handle case where capability isn't supported */ }
+```
+
+For more information about the install dialog see, the [appInstallDialog.openAppInstallDialog()](/javascript/api/@microsoft/teams-js/appinstalldialog?view=msteams-client-js-latest#@microsoft-teams-js-appinstalldialog-openappinstalldialog&preserve-view=true) function in the API reference documentation.
+
+# [TeamsJS v1](#tab/teamsjs-v1)
+
+```javascript
 // Open an app install dialog from your tab
 microsoftTeams.executeDeepLink("https://teams.microsoft.com/l/app/f46ad259-0fe5-4f12-872d-c737b174bcb4");
 ```
 
-## Deep linking to a chat
+---
+
+### Deep link to mail
```

Comment:
Mail isn't deep links at all, but I can see how it got here since it is conceptually similar to things like calendar. *no* deep links are generated for mail in any scenarios, and mail did not exist in the 1.x sdk. I think we should remove the mail section from this page if this is specifically about how deep linking has changed in 2.x

Reply:
Removed. Will work to figure out where this should live in the longer term soon.